### PR TITLE
Upgrade k8s version to 1.19

### PR DESCRIPTION
**Pull request by tei-k**

### CODE REVIEW

File: modules/azure/kubernetes/locals.tf [hunk 1]
```diff
@@ -27,11 +27,11 @@ locals {
     resource_group_name       = var.network.name
     region                    = var.network.region
     dns_prefix                = "scalar-kubernetes"
-    kubernetes_version        = "1.16.13"
+    kubernetes_version        = "1.19.7"
     admin_username            = "azureuser"
     public_ssh_key_path       = var.network.public_key_path
     role_based_access_control = true
-    kube_dashboard            = true
+    kube_dashboard            = false
```
Comment (feeblefakie) [inline modules/azure/kubernetes/locals.tf:34]: Is this change from some limitation?
If so, we should note it here as a comment.

**Review by feeblefakie — COMMENTED** [review]
Left one question.

File: modules/azure/kubernetes/locals.tf (hunk 1 above)
Comment (tei-k) [reply modules/azure/kubernetes/locals.tf]: Because kube_dashboard add-on is deprecated for versions 1.19+, so deleted this option in e3c0d77.

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Comment by feeblefakie** [comment]
@ymorimo Merging it but PTAL when you get a chance.

**Review by ymorimo — COMMENTED** [review]
LGTM! Sorry for the late review.

**Pull request merged**